### PR TITLE
improve comments in lxqtsingleapplication.h

### PR DESCRIPTION
**Pull request by ianorlin**
I have improved the comment in lxqtsingleapplication.h and made them easier to read for future contributors.

### CODE REVIEW

**Comment by paulolieuthier** [comment]
Please squash the commits.

File: lxqtsingleapplication.h [hunk 1]
```diff
@@ -60,7 +60,7 @@ namespace LxQt {
  *      return app.exec();
  *  }
  *
- *  // SingleApplication code
+ *  // Singleore utility library for all LXQt components
```
Comment (palinek) [inline lxqtsingleapplication.h]: >singleore

what's the meaning of that word?

**Comment by paulolieuthier** [comment]
@ianorlin?

**Comment by paulolieuthier** [comment]
Fixed and merged in 355d2860ad782e3b9d82441d05e78f9ed239a7c7.

**Pull request closed without merging**